tasks/clean.yml

```diff
@@ -18,3 +18,9 @@
   apt_repository:
     repo: deb http://packages.treasuredata.com/4/ubuntu/{{ ansible_distribution_release|string }}/ {{ ansible_distribution_release|string }} contrib
     state: absent
+
+- name: Fluentd | Remove td-agent user
```

Comment:
why do we remove this?

Reply:
If you do not delete the user, then the process works on his behalf, even if another user is specified in fluentd.service, restarts do not help. This may lead to a conflict of directory and process rights between _fluent and td-agent users.